script/run-docker

```diff
@@ -0,0 +1,106 @@
+#!/bin/sh
+
+# Source the `config/environment` script.
+#
+# TODO This should be removed?  Ideally, if the two environment
+# variables are set, we should be all good to go.
+. $(dirname $0)/../config/environment
```

Comment:
I'd like to think about replacing this with either default values, or exiting with an error if the env vars aren't defined. 

I agree with not really wanting the `config/` dir 😄